sound/soc/intel/boards/sof_sdw_common.h

```diff
@@ -79,6 +79,24 @@ enum {
 
 #define SOF_SDW_MAX_DAI_NUM		8
 
+static const struct snd_soc_dapm_widget generic_spk_widgets[] = {
```

Comment:
typo in commit message: s/wiegets/widgets

Reply:
> typo in commit message: s/wiegets/widgets

fixed